stl/inc/xutility

```diff
@@ -2005,6 +2004,20 @@ _NODISCARD constexpr size_t size(const _Ty (&)[_Size]) noexcept {
     return _Size;
 }
 
+#if __HAS_CXX20
+template <class _Container>
+[[nodiscard]] constexpr auto ssize(const _Container& _Cont)
+    -> common_type_t<ptrdiff_t, make_signed_t<decltype(_Cont.size())>> {
+    using _rTy = common_type_t<ptrdiff_t, make_signed_t<decltype(_Cont.size())>>;

```

Comment:
We conventionally avoid "underscore-lowercase" identifiers in the STL and use them only in the C library and/or very low-level machinery that needs to be callable from both C and C++. There's no particular reason to avoid them outside of the global namespace, other than it's convenient to know that when you see such a name some scary juju is happening.

Please rename `_rTy` to something "underscore-capital" - I suggest `_RTy` or `_Result`.

Reply:
Switched to `_Common`